en/docs/updates/migrating-to-updates2.0.md

```diff
@@ -0,0 +1,50 @@
+### How do I migrate from WUM and In-place to Updates 2.0?
+
+* Update the current product pack to the latest update level using WUM or in-place tool. This will make Update tool
+  client available inside `<product_home>/bin` directory.
+* Run the [Update commands](../../updates/update-tool/) on top of the update tool. This will
+  update the product pack to comply with Updates 2.0
+
+### How can a user merge the *WUM Diff* manually to get updates using WSO2 Updates 2.0?
+To get Updates using the new WSO2 Updates model follow the steps below<br>
+
+1. First get the newest product pack from WUM (Run `wum update <product>` e.g., wum update wso2am-3.0.0). <br>
+2. Apply [WUM diff](https://docs.wso2.com/display/updates100/WUM+Commands+Guide#WUMCommandsGuide-wumdiff) as you have been doing before to a copy of product pack (which contain your customizations). <br>
+
+!!! Note
+    The diff ZIP is created in the location from where you executed the diff command.
+<br>
+3. Now your current pack is congruent with WSO2 Update 2.0, thus you are able to run the [Updates tool](../../updates/update-tool/) and receive WSO2 Updates.<br>
+4. Test the updated product distribution and deploy it in your production environment.
+
+### Is it mandatory to have a configuration management tool configured or can we take the Updates manually?
+It is not mandatory to have a configuration management tool configured, but having a configuration management tool makes the continuous updates process easy, 
+and it aids configuration with less possibility of human error.
+
+### what’s are the correct steps to apply the same Update level to all the nodes in the deployment if the user has not configured a configuration management tool?
```

Comment:
Shall we change this to `What are the correct steps to apply the same Update level to all the nodes in the deployment if a configuration management tool is not being used?`?